settings.gradle

```diff
@@ -11,6 +11,7 @@ include ':extensions:android-paging3'
 include ':extensions:coroutines-extensions'
 include ':extensions:rxjava2-extensions'
 include ':extensions:rxjava3-extensions'
+include ':postgresql-dialect'
```

Comment:
lets create a new `postgres` and name this piece of code `runtime` since its the postgres-specific version of the sqldelight runtime, so it would be `postgres:runtime`

Reply:
Love it.

Reply:
Do we want our coordinates to look like this

```
app.cash.sqldelight.postgresql:runtime:2.0.0
```

or this 

```
app.cash.sqldelight:postgresql-runtime:2.0.0
```

I'm running into some name clashes with the former but that can be overcome by fiddling with module groupIds. Want to double check that it's worth the effort before I do it.

Reply:
second one is fine for now, if we move it to a separate repo we can change it